README.md

```diff
@@ -37,4 +38,4 @@ Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
```

Comment:
https://github.com/Amedzro-Elikplim/School-Library/blob/03fe0a34d84e81c48fbe4386cf2b8d4b3535350c/main.rb#L13-L15

- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, Linter is a tool for identifying and reporting errors when you use standard rules. Kindly get used to fix liner errors manually instead of disabling it 👍🏻